pkg/operator/ceph/object/objectstore.go

```diff
@@ -99,59 +105,79 @@ func reconcileRealm(context *Context, serviceIP string, port int32) error {
 		defaultArg = "--default"
 	}
 
-	// create the realm if it doesn't exist yet
-	output, err := runAdminCommand(context, "realm", "get")
-	if err != nil {
+	// obviously need to change this
+	if isMultisite(spec) {
 		updatePeriod = true
-		output, err = runAdminCommand(context, "realm", "create", defaultArg)
+		realmID := spec.Multisite.Realm
+		realmArg = fmt.Sprintf("--rgw-realm=%s", realmID)
+		zoneGroupID := spec.Multisite.ZoneGroup
+		zoneGroupArg = fmt.Sprintf("--rgw-zonegroup=%s", zoneGroupID)
+		zoneID := spec.Multisite.Zone
```

Comment:
nit: instead of assigning the ream, zonegroup, and zone to local variables, just use them directly where needed.